mpd/fixtures/live_profile.mpd

```diff
@@ -2,22 +2,22 @@
 <MPD xmlns="urn:mpeg:dash:schema:mpd:2011" profiles="urn:mpeg:dash:profile:isoff-live:2011" type="static" mediaPresentationDuration="PT6M16S" minBufferTime="PT1.97S">
   <Period>
     <AdaptationSet mimeType="audio/mp4" segmentAlignment="true" startWithSAP="1" lang="en">
-      <ContentProtection schemeIdUri="urn:mpeg:dash:mp4protection:2011" cenc:default_KID="08e36702-8f33-436c-a5dd60ffe5571e60" value="cenc" xmlns:cenc="urn:mpeg:cenc:2013"></ContentProtection>
```

Comment:
Our tests don't check XML equivalency just string equality, so the order of the attributes matter. I'll add an issue.